src/dropdowns/dropdown-trigger.ts

```diff
@@ -0,0 +1,46 @@
+import {Directive, HostListener, ElementRef} from 'angular2/core';
+import {NglDropdown} from './dropdown';
+
+@Directive({
+  selector: '[nglDropdownTrigger]',
+  host: {
+    'aria-haspopup': 'true',
+    'tabindex': '0',
```

Comment:
It seems that for this case, even when `nglDropdownItem` is used correctly on the `<a>`, without a tabindex attribute focusing on the elements is not possible.